lib/utils.js

```diff
@@ -5,19 +5,38 @@ const exec = require('child_process').exec;
 const qs = require('qs');
 const crypto = require('crypto');
 
-const hasOwn = {}.hasOwnProperty;
-const isPlainObject = require('lodash.isplainobject');
+const hasOwn = Function.bind.call(
+  Function.call,
+  Object.prototype.hasOwnProperty
+);
```

Comment:
Just an extra gift, to avoid calling `hasOwn.call` and just do `hasOwn(`

Reply:
Now that we're in ES6, mind clarifying to `const hasOwn = (obj, key) => {}.hasOwnProperty.call(ob, key)` ? Just a bit clearer what's going on that way.